tensorrt_llm/executor/rpc_proxy.py

```
@@ -115,6 +115,17 @@ def get_stats(self, timeout: float) -> List[dict]:
             logger.debug(f"Error fetching stats via RPC: {e}")
             return []
 
+    def get_kv_cache_capacity(self) -> List[dict]:
+        """Get static KV cache capacity from the runtime via RPC."""
+        try:
+            capacity = self.rpc_client.fetch_kv_cache_capacity_async().remote()
+            return [
+                json.loads(c) if isinstance(c, str) else c for c in capacity
+            ]
+        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)